ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorDemoController.swift

```diff
@@ -105,33 +105,42 @@ class ColorDemoController: UIViewController {
         tableView.dataSource = self
         tableView.delegate = self
         tableView.separatorStyle = .none
+        tableView.allowsSelection = false
+        tableView.backgroundColor = Colors.Table.background
 
-        let stackView = UIStackView(arrangedSubviews: [segmentedControl, tableView])
+        let separator = Separator(style: .shadow, orientation: .horizontal)
+        let stackView = UIStackView(arrangedSubviews: [segmentedControl, separator, tableView])
+        stackView.setCustomSpacing(8, after: segmentedControl)
         stackView.axis = .vertical
         stackView.translatesAutoresizingMaskIntoConstraints = false
 
         view = UIView(frame: .zero)
         view.addSubview(stackView)
-        view.backgroundColor = Colors.Table.background
-
-        let safeAreaLayoutGuide = view.safeAreaLayoutGuide
+        view.backgroundColor = Colors.NavigationBar.background
 
         NSLayoutConstraint.activate([
-            stackView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor),
-            stackView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor),
-            stackView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor),
-            stackView.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor)
+            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
```

Comment:
Did you test this on a notched iPhone in landscape?

Reply:
yeah I did. this entire view controller seems not right to host the tableview inside a stack. because the cell should go all the way edge to edge but the content inset should be within safe ware bound.